.docker-compose.datagov-catalog.yaml

```diff
@@ -0,0 +1,54 @@
+version: '3.2'
```

Comment:
Please modify the  `.docker-compose.datagov.yaml` for modifying the base image and don't create a new base image flavour..

Reply:
I will use the previous yaml file but I'm thinking that we already need two flavours: Inventory and Catalog